package.json

```diff
@@ -23,7 +23,8 @@
     "Other"
   ],
   "activationEvents": [
-    "workspaceContains:*.py"
+    "onLanguage:python",
+    "onStartupFinished"
```

Comment:
Think we should only activate on python files so as not to use resources. I did a PR for this before https://github.com/sourcery-ai/sourcery-vscode/pull/13

Reply:
Then the clone detection doesn't work until you've opened a python file